shadows/framework/src/main/java/org/robolectric/shadows/ShadowLauncherApps.java

```diff
@@ -39,6 +41,7 @@
 @Implements(value = LauncherApps.class, minSdk = LOLLIPOP)
 public class ShadowLauncherApps {
   private List<ShortcutInfo> shortcuts = new ArrayList<>();
+  private Map<UserHandle, List<String>> enabledPackages = new HashMap<>();
```

Comment:
This may be slightly simpler using a `Multimap<UserHandle, String>`:

```java
private Multimap<UserHandle, String> enabledPackages = HashMultimap.create();
```

Then both addEnabledPackage and isPackageEnabled become one-liners.